scalanet/src/io/iohk/scalanet/reactive/ObservableOps.scala

```diff
@@ -0,0 +1,42 @@
+package io.iohk.scalanet.reactive
+
+import monix.execution.Scheduler
+import monix.reactive.Observable
+
+object ObservableOps {
+
+  implicit class ObservableExtension[T](o: Observable[T]) {
+
+    /**
+      * Split an observable by a predicate, placing values for which the predicate returns true
+      * to the right (and values for which the predicate returns false to the left).
+      * This is consistent with the convention adopted by Either.cond.
+      */
+    def split(
+        p: T => Boolean
+    )(implicit scheduler: Scheduler): (Observable[T], Observable[T]) = {
+      splitEither[T, T](elem => Either.cond(p(elem), elem, elem))
```

Comment:
I think that just `(o.filter(p), o.filterNot(p))` would be simpler and more efficient, as it doesn't involve creating new object before filtering.